addons/bestja_frontend_fpbz/templates/pages/home_page.xml

```diff
@@ -1,398 +1,400 @@
+                    <section class="oe_snippet_body">
+                        <div class="container-fluid">
+                            <div class="row">
+                                <div class="col-md-12">
+                                    <h1 class="text-center mt48 mb48 section_title">
+                                        <strong>Wybierz</strong> gdzie chciałbyś pomagać</h1>
+                                </div>
+                            </div>
+                        </div>
+                    </section>
+                    <section class="oe_snippet_body">
+                        <div id="map_of_poland_section" class="container-fluid map_of_poland_section">
+                            <div class="container-fluid">
+                                <div class="row">                                    
+                                    <div class="col-md-7 mt16 map_of_poland_container">
+                                        <object class="map_object" 
+                                                data="/bestja_frontend_fpbz/static/src/img/content/wolontariat-mapa-02.svg" type="image/svg+xml">
+                                        </object>
+                                    </div>
+                                    <div class="col-md-4 mt16 description_of_map_container">
+                                        <div>
+                                            <p>Twoja organizacja chciałaby z nami współpracować?</p>
+                                            <p>
+                                                <a class="color_orange">
+                                                    <strong>Dowiedz się więcej</strong>
+                                                </a> lub 
+                                                <a class="color_orange">
+                                                    <strong>zostań organizacją partnerską</strong>
+                                                </a>
+                                            </p>
+                                        </div>
+                                    </div>
+                                </div>
+                            </div>
+                        </div>
+                    </section>
+                    <section class="oe_snippet_body">
+                        <div class="container-fluid quote_div">
+                            <div class="row">
+                                <div class="col-md-offset-1 col-md-10">
+                                    <blockquote>Wolontariat daje mi 
+                                        przede wszystkim satysfakcję z pomagania, 
+                                        a także możliwość kreatywnego działania. Poznaję nowych ludzi, 
+                                        uczę się pracy w zespole i rozwijam swoje zainteresowania.
+                                    </blockquote>
+                                    <span class="podpis_pod_cytatem">

```

Comment:
This is a regression, the class `podpis_pod_cytatem` has already been replaced by its English version.